src/js/tabs/firmware_flasher.js

```diff
@@ -107,10 +107,19 @@ firmware_flasher.initialize = function (callback) {
             $('div.release_info #targetMCU').text(summary.mcu);
             $('div.release_info .configFilename').text(self.isConfigLocal ? self.configFilename : "[default]");
 
-            // Wiki link to url found in unified target configuration or if not defined to general wiki url
+            // Wiki link: #wiki found in unified target configuration, if board description exist or generel board missing
+            let urlWiki = 'https://betaflight.com/docs/wiki/boards/missing';                // generel board missing
+            const urlBoard = `https://betaflight.com/docs/wiki/boards/${summary.target}`;   // board description
+            if (summary.wiki === undefined) {                                               // override if #wiki exist in unified target configuration
+                if (urlExists(urlBoard)) {
+                    urlWiki = urlBoard;
+                }
+            } else {
+                urlWiki = summary.wiki;
+            }
```

Comment:
```suggestion
            if (urlExists(urlBoard)) {
                urlWiki = urlBoard;
            }
```

Reply:
I think we should keep the overide possibility, we can ue this to point more config files to one single documentaion, if needed
(I will stop working for today now)

Reply:
The url does not work now with KAKUTEH7 without this change.

Rather we use target name and default path in documentation and remove the dependency on unified targets repo reducing maintenance otherwise we need to update config.h too.

Reply:
Only KAKUTEF4 has the right #wiki string , the others are from before betaflight website was redesigned.

My suggestion are like you want, a default to (2.) if board description exist in https://betaflight.com/docs/wiki/boards/ + target name
But still keep the override possibility, can solve fx if we want to point to documentation in archive for old boards, and we already have the logic in summary struct

Reply:
The other reason we should not use summary.wiki is that firmware 4.5 does not use unified targets configuration.
As we already have the target board name itself there is no need to explicit use a configuration file for a pointer to documentation. Much easier to create 4 pages for the variants [for the ALIEANFLIGHTF7] and be done.

Reply:
OK, now I get it, wiki way obsolute in firmware 4.5